### DISCUSSION TITLE
[Feature] Partner sharing since date...

### DISCUSSION THREAD

**Discussion by swex** [immich-app/immich#10558] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

There are many possible use cases... 
Google Photos has such functionality.
PS: 
Not only since, but maybe date range may be useful.
Filter by position also would be great! 

### Platform

- [X] Server
- [ ] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
#2654
